CONTRIBUTING.md

```diff
@@ -44,6 +44,12 @@ GitHub provides additional document on [forking a repository](https://help.githu
 Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels (enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/awslabs/aws-lambda-builders/labels/help%20wanted) issues is a great place to start. 
 
 
+## Testing with SAM CLI
+
+1. Activate Virtualenv that is used for SAM CLI development.
```

Comment:
Lets link to development guide here: https://github.com/awslabs/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.md

without that the virtualenv to be used is not known.

Reply:
Added a link to the SAM CLI's Development Guide.